core/templates/pages/library-page/library-page.directive.html

```diff
@@ -55,7 +55,7 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
             <h2 class="oppia-classroom-h2" translate="I18N_CLASSROOM_CALLOUT_HEADING_2"></h2>
             <h1 class="oppia-classroom-h1" translate="I18N_CLASSROOM_CALLOUT_HEADING_1"></h1>
             <p class="oppia-classroom-p" translate="I18N_CLASSROOM_CALLOUT_PARAGRAPH_1"></p>
-            <a class="btn oppia-dashboard-intro-button oppia-transition-200" translate="I18N_CLASSROOM_CALLOUT_BUTTON" ng-href="/classroom/Math" href="/classroom/Math"></a>
+            <a class="btn oppia-dashboard-intro-button oppia-transition-200" translate="I18N_CLASSROOM_CALLOUT_BUTTON" ng-href="/Math" href="/Math"></a>
```

Comment:
Same thing re capitalization.

Reply:
Discussion for capitalization in the comment [above](https://github.com/oppia/oppia/pull/9109#discussion_r410668710).